app/src/main/java/com/example/dndwiki/network/FakeNetwork.kt

```diff
@@ -1,14 +1,14 @@
 package com.example.dndwiki.network
 
-import androidx.lifecycle.MutableLiveData
 import com.example.dndwiki.data.DamageType
 import com.example.dndwiki.data.School
 import com.example.dndwiki.data.Spell
 
 class FakeNetwork {
 
     private var spells = listOf(
```

Comment:
You can make this a public val  and you woudn't need the getSpells function anymore